docs/csharp/programming-guide/concepts/serialization/index.md

```diff
@@ -12,57 +12,69 @@ This illustration shows the overall process of serialization:
 
 ![Serialization graphic](./media/index/serialization-process.gif)
 
-The object is serialized to a stream, which carries not just the data, but information about the object's type, such as its version, culture, and assembly name. From that stream, it can be stored in a database, a file, or memory.
+The object is serialized to a stream that carries the data. The stream may also have information about the object's type, such as its version, culture, and assembly name. From that stream, the object can be stored in a database, a file, or memory.
 
 ### Uses for serialization
 
-Serialization allows the developer to save the state of an object and recreate it as needed, providing storage of objects as well as data exchange. Through serialization, a developer can perform actions like sending the object to a remote application by means of a Web Service, passing an object from one domain to another, passing an object through a firewall as an XML string, or maintaining security or user-specific information across applications.
+Serialization allows the developer to save the state of an object and re-create it as needed, providing storage of objects as well as data exchange. Through serialization, a developer can perform actions such as:
 
-### Making an object serializable
+* Sending the object to a remote application by using a web service
+* Passing an object from one domain to another
+* Passing an object through a firewall as a JSON or XML string
+* Maintaining security or user-specific information across applications
 
-To serialize an object, you need the object to be serialized, a stream to contain the serialized object, and a <xref:System.Runtime.Serialization.Formatter>. <xref:System.Runtime.Serialization> contains the classes necessary for serializing and deserializing objects.
+## JSON serialization
 
-Apply the <xref:System.SerializableAttribute> attribute to a type to indicate that instances of this type can be serialized. An  exception is thrown if you attempt to serialize but the type doesn't have the <xref:System.SerializableAttribute> attribute.
+The [System.Text.Json namespace](../../../../standard/serialization/system-text-json-overview.md) contains classes for JavaScript Object Notation (JSON) serialization and deserialization. JSON is an open standard that is commonly used for sharing data across the web.
```

Comment:
Shouldn't this link to the API ref instead of the overview doc?

Reply:
For this doc's audience -- people new to serialization -- it seems to me that the overview doc is more useful than the namespace API ref. The link text does make it look like you'd get an API ref doc, though, so I can see why that might be a good choice as well.

Reply:
Other links point to the API ref. The namespace should point to the API imo. Maybe you can add a sentence on how to use it and have that link to the how-to doc.